### DISCUSSION TITLE
Visual Studio: Copilot extension version 1.93 slows down VS significantly

### DISCUSSION THREAD

**Discussion by laicasaane** [community/community#59609] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Today I have just upgrade GitHub Copilot extension on Visual Studio to 1.93 and observe that the whole VS is gradually slowing down and sometimes freezing for about 2 seconds. The CPU usage of VS is really high, sometimes reaches 80%.

When I disable the extension, VS becomes normal again.

### Information

Visual Studio: 17.6.4

**Comment by laicasaane** [comment]
Could I reinstall the previous version of the extension? There seems to be no way to get a specific version from the extension marketplace.

**Comment by wallace-sf** [comment]
I'm facing the same issue. Copilot is taking several seconds to show code hints. In the previous version, i had no problems with that.

PC Specs:

Processor: Intel(R) Core(TM) i7-10510U CPU @ 1.80GHzGraphics: NVIDIA GeForce MX250
Memory: 16 GB DDR4 RAM
Operating System: Ubuntu 20.04.6 LTS

VS Code:

Version: 1.80.1
Commit: 74f6148eb9ea00507ec113ec51c489d6ffb4b771
Date: 2023-07-12T17:22:25.257Z
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 5.15.0-76-generic snap

**Comment by airtonix** [comment]
```
Version: 1.78.2
Commit: b3e4e68a0bc097f0ae7907b217c1119af9e03435
Date: 2023-05-10T14:47:05.613Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.2.15-200.fc37.x86_64
Sandboxed: Yes
```

```
Name: GitHub Copilot
Id: GitHub.copilot
Description: Your AI pair programmer
Version: 1.96.263
Publisher: GitHub
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=GitHub.copilot
```

Still makes my cpu AND GPU fans spin up.

Looks like we still have to install the older `1.8.x` series.


@laicasaane

[Screencast from 2023-07-17 16-04-46.webm](https://github.com/community/community/assets/61225/c6c1ad56-407d-409b-bef3-f77518a6711e)

**Reply by ayim** [reply]
This looks like Visual Studio Code, I recommend you file a separate issue.

**Comment by rookledookle** [comment]
Same issue here - unusable in this state

**Reply by ayim** [reply]
@rookledookle if you're experiencing an issue in Visual Studio with Copilot extension, I recommend filing an issue using Visual Studio's built in "Report a Problem" feature so that we can get more diagnostics information to triage and debug.

![image](https://github.com/community/community/assets/10951869/9f03e0df-40ee-428c-baaa-f299aeb9b73b)


--
Aaron Yim
Product Manage for GitHub Copilot in VS
aayim@microsoft.com

**Comment by ayim** [comment]
Locking conversation since discussion RE: VSCode is stale at this point.
Instructions for reporting new Visual Studio bugs also noted above.
